README

```diff
@@ -55,13 +55,17 @@ and two required arguments:
 handshake [-t] [-s] <certsdir> <threadcount>
 -t - produce terse output
 -s - create an ssl_ctx per connection, rather than a single thread-shared ctx
+-p - use ossl_lib_ctx per thread
 certsdir - Directory where the test can locate servercert.pem and serverkey.pem
 threadcount - Number of concurrent threads to run in test
 
 The output is two values: the average time taken for a single handshake in us,
 and the average number of simultaneous handshakes per second performed over the
 course of the test.
 
+Note: ossl_lib_ctx per thread with higher number of threads will hit the thread
```

Comment:
I don't know how others feel about it. In my opinion we should not be adding such note here. I think the thread local storage issue should get fixed rather quickly.

depending on whether the fix to libcrypto will be backported to release branches we might want to have such note saying something like: `Note on OpenSSL earlier than ?3.6? you might hit issue ....`

Reply:
Ok, I'll drop it.